packages/commerce-sdk-react/src/auth/storage.ts

```diff
@@ -77,26 +75,26 @@ export class LocalStorage extends BaseStorage {
         }
     }
     set(key: string, value: string) {
-        const prefixedKey = this.getPrefixedKey(key)
-        const oldValue = this.get(prefixedKey)
-        window.localStorage.setItem(prefixedKey, value)
+        const suffixedKey = this.getSuffixedKey(key)
```

Comment:
Correct me if I am wrong, it looks like we are replace prefix with suffix for all the keys in storage? I thought originally we only want to do it for site_id key, not every key.

Reply:
there is only 1 use case for suffix and it is the siteId.

Reply:
I get that for the side_id use case, but I was talking about if we add another key,  with the changes here means it will apply for any key that need to have suffix? 🤔 , so basically we move from using prefix to suffix for any key that uses it?

Reply:
that's right... unless there is a use case for prefix, this change moves from using prefix to suffix for **everything**.